ngraph/core/src/op/convert.cpp

```diff
@@ -34,19 +34,30 @@ op::Convert::Convert(const Output<Node>& arg, const element::Type& destination_t
 
 void op::Convert::validate_and_infer_types()
 {
-    set_output_type(0, m_destination_type, get_input_partial_shape(0));
+    NGRAPH_OP_SCOPE(Convert_validate_and_infer_types)
+    {
+        set_output_type(0, m_destination_type, get_input_partial_shape(0));
+    }
 }
```

Comment:
Maybe fail check otherwise?

Reply:
Let's decide it. I didn't throw any exception. If we want to fail, I also will throw an exception from all disabled methods.
But at the current moment I don't throw exceptions in order to avoid situation that we throw an exception but user doesn't expects it.

Reply:
It will be easier to debug possible problems with CC if error will be generated here.

Reply:
Ok, I will add exceptions

Reply:
Fixed